uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/CopyPopupViewImpl.java

```diff
@@ -85,6 +99,9 @@ public String getCheckInComment() {
 
     @Override
     public void handleInvalidFileName( String baseFileName ) {
-        Window.alert( CommonConstants.INSTANCE.InvalidFileName0( baseFileName ) );
+        nameFormStyleItem.getFormGroup().setValidationState( ValidationState.ERROR );
+
+        // TODO : Change text of the validation message to "Invalid plugin name."
```

Comment:
Following our discussion this is correct if we're to have consistency across all rename/copy actions for all asset types.